kotlin-native/litmus/results/index.html

```diff
@@ -0,0 +1,128 @@
+
```

Comment:
Do we need to include generated report to the repo? 
Perhaps it would make sense, once we will have the stable test suite.